torchbenchmark/util/framework/huggingface/model_factory.py

```diff
@@ -1,5 +1,8 @@
+import math
+import random
 import torch
 from torch import optim
+from torch.jit.frontend import NotSupportedError
```

Comment:
Can we import this in `get_dynamic_shaps_module` as well, since it is not used in eager mode?